canonical-web-and-design/ubuntu.com
# Update text, data and one calculation on /openstack/tco-calculator

## Done

- Updated /openstack/tco-calculator based on pricing changes
- Includes a change in calculations for managed services

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/tco-calculator
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare the text chage to the [copy doc](https://docs.google.com/document/d/1n3ExZwFVlw5LyAp7O3CgPthVgx8yRX5-Wg_gBQLf73c/edit#)
- Compare the calc changes to the [google sheet](https://docs.google.com/spreadsheets/d/1qOFmJ2zOy0MNqANk1IQnrBgOPhLxwoj7g8Nvyl4y_Ic/edit#gid=0)

@tytus-kurek can you check the calculations?

## Issue / Card

Fixes #9244
